--- contrib/mc-wrapper.csh.in~	2017-02-24 14:25:57.000000000 -0600
+++ contrib/mc-wrapper.csh.in	2017-05-17 07:39:06.830108548 -0500
@@ -3,7 +3,7 @@
 if ($?TMPDIR) then
 	setenv MC_PWD_FILE $TMPDIR/mc-$MC_USER/mc.pwd.$$
 else
-	setenv MC_PWD_FILE /tmp/mc-$MC_USER/mc.pwd.$$
+	setenv MC_PWD_FILE /var/tmp/mc-$MC_USER/mc.pwd.$$
 endif
 
 @bindir@/mc -P "$MC_PWD_FILE" $*
--- contrib/mc-wrapper.sh.in~	2017-02-24 14:25:57.000000000 -0600
+++ contrib/mc-wrapper.sh.in	2017-05-17 07:38:48.348215251 -0500
@@ -1,5 +1,5 @@
 MC_USER=`id | sed 's/[^(]*(//;s/).*//'`
-MC_PWD_FILE="${TMPDIR-/tmp}/mc-$MC_USER/mc.pwd.$$"
+MC_PWD_FILE="${TMPDIR-/var/tmp}/mc-$MC_USER/mc.pwd.$$"
 @bindir@/mc -P "$MC_PWD_FILE" "$@"
 
 if test -r "$MC_PWD_FILE"; then
